File: TwoSum.java
public class TwoSum {
    public static void main(String[] args) {
        int [] lst={7, 4, 5, 6, 1,2, 8, 9};
        int target=8;
        for(int i=0;i<lst.length;i++){
            for(int j=0;j<lst.length;){
                if(lst[i]+lst[j]==target){
                    System.out.println(i+" "+j);
                    
                }
                break;
            }

        }
    }
}

File: TwoSum.py
# lst = [7, 4, 5, 6, 1,2, 8, 9]
lst=[1,2,3,4]
target = 7

for i in lst:
    for j in lst:
        if i + j == target:
            print(lst.index(i), lst.index(j))
            break

